modules/UI/videolayout/LocalVideo.js

```diff
@@ -24,6 +24,9 @@ function LocalVideo(VideoLayout, emitter) {
         this._buildContextMenu();
     this.isLocal = true;
     this.emitter = emitter;
+    this.statsPopoverLocation = interfaceConfig.VERTICAL_FILMSTRIP
+        ? 'left bottom' : 'top right';
```

Comment:
InlineDialog/popper does not ensure it displays itself fully in the viewport. With the local video on the bottom right, it is possible for stats to them display cut off on the bottom when fully expanded. To make up for this, nudge it up (which is "bottom" due to other css tricks in play).